src/main/java/com/nomiceu/nomilabs/event/CommonProxy.java

```diff
@@ -95,4 +99,17 @@ public static void syncConfigValues(ConfigChangedEvent.OnConfigChangedEvent even
             ConfigManager.sync(LabsValues.LABS_MODID, Config.Type.INSTANCE);
         }
     }
+
+    @SubscribeEvent
+    public static void missingMappings(MissingMappings<Item> event) {
+        for (MissingMappings.Mapping<Item> entry : event.getAllMappings()) {
+            // Remap old DevTech perfect gem to new perfect gem.
+            // DevTech did this badly, and created a MetaPrefixItem with GT's material registry but their
+            // Mod ID, so we need to map the DevTech metaitem to the GT metaitem in missing mappings.
+            if (entry.key.toString().equals("devtech:meta_gem_perfect")) {
+                ResourceLocation newMapping = new ResourceLocation(GTValues.MODID, "meta_gem_perfect");
```

Comment:
No, GT handles prefix items by namespacing them by the registry. DevTech's was using the GT material registry with its own modid, when it always needed to be the GT modid. Since GT and each addon have their own material registries (potentially), this will cause problems if done another way (plus, using nomilabs name here wouldn't work anyway, since nomilabs doesnt have a Diamond, Emerald, Ruby, etc material in its material registry, so there is nothing to remap to)